app/views/service_providers/edit.html.slim

```diff
@@ -1,7 +1,8 @@
 h2 = t('headings.service_providers.edit')
 
 = simple_form_for(service_provider, html: { autocomplete: 'off', role: 'form' }) do |form|
-  == render 'form', form: form
+  - selected_user_group = @service_provider.user_group_id || nil
```

Comment:
you can just write this as `@service_provider.user_group_id` because that will return `nil` if it is `nil` anyway